src/main/java/EvgenUlianov/HomeTask_1_1/Security/CustomPasswordEncoder.java

```diff
@@ -0,0 +1,12 @@
+package EvgenUlianov.HomeTask_1_1.Security;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CustomPasswordEncoder extends BCryptPasswordEncoder implements PasswordEncoder {
```

Comment:
А зачем он нужен, если не делает ничего отличного от `BCryptPasswordEncoder`?

Reply:
без этого класса не работает (ошибка: Parameter 1 of constructor in EvgenUlianov.HomeTask_1_1.UserManager.UserService required a bean of type 'org.springframework.security.crypto.password.PasswordEncoder' that could not be found. ), мне нужна имплементация "PasswordEncoder" с аннотацией "@Component",
этот интерфейс у меня используется в "WebSecurityConfig" и "UserService"
(этот класс у меня родился в процессе отладки)

Reply:
Можно было прямо в классе конфигурации сделать
```suggestion
@Bean
public PasswordEncoder encoder() {
    return new BCryptPasswordEncoder();
}
```